modelon/impact/client/experiment_definition.py

```diff
@@ -962,7 +992,9 @@ def initialize_from(self, entity):
             self._simulation_options,
             self._simulation_log_level,
         )
-        _validate_and_set_initialize_from(entity, new)
+        _validate_and_set_initialize_from(
+            entity, new, [entities.Experiment, entities.Case]
+        )
```

Comment:
Instead of being able to send in what entities are supported (which is a bit complicated) could you not just add this code here:
```
if isinstance(entity, entities.ExternalResult):
  raise TypeError('It is not supported to specify initialize from external result for experiment extensions')
```